PROTOCOL.md

```diff
@@ -0,0 +1,193 @@
+# Twirp Wire Protocol
+
+This document defines the Twirp wire protocol over HTTP.
+
+## Conventions
+
+The requirement level keywords "MUST", "MUST NOT", "REQUIRED",
+"SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY",
+and "OPTIONAL" used in this document are to be interpreted as
+described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The grammar rules used in this document are using [ABNF
```

Comment:
We only use ABNF in one place where it's useful, which is in describing the URL. Let's delete this paragraph and just say something like `In [ABNF syntax](https://tools.ietf.org/html/rfc5234), Twirp's URLs follow this format: ...`

Then, we can get rid of this `Conventions` section altogether, which is good.